aws/resource_aws_route53_resolver_dnssec_config_test.go

```diff
@@ -178,34 +167,21 @@ func testAccCheckRoute53ResolverDnssecConfigExists(n string, config *route53reso
 			return fmt.Errorf("No Route 53 Resolver Dnssec config ID is set")
 		}
 
+		id := rs.Primary.ID
 		conn := testAccProvider.Meta().(*AWSClient).route53resolverconn
-		input := &route53resolver.ListResolverDnssecConfigsInput{}
-
-		err := conn.ListResolverDnssecConfigsPages(input, func(page *route53resolver.ListResolverDnssecConfigsOutput, lastPage bool) bool {
-			if page == nil {
-				return !lastPage
-			}
-
-			for _, c := range page.ResolverDnssecConfigs {
-				if aws.StringValue(c.Id) == rs.Primary.ID {
-					config = c
-					return false
-				}
-			}
 
-			return !lastPage
-		})
+		config, err := finder.ResolverDnssecConfigByID(conn, id)
 
 		if err != nil {
 			return err
 		}
 
 		if config == nil {
-			return fmt.Errorf("No Route 53 Resolver Dnssec config found")
+			return fmt.Errorf("Route53 Resolver Dnssec config (%s) not found", id)
```

Comment:
Super nit 😭 : Route 53 does in fact have a space: https://aws.amazon.com/route53/

```suggestion
			return fmt.Errorf("Route 53 Resolver Dnssec config (%s) not found", id)
```